.ci/setup.sh

```diff
@@ -14,6 +14,7 @@ source "${cidir}/lib.sh"
 apply_depends_on
 
 arch=$(arch)
+INSTALL_KATA="${INSTALL_KATA:-TRUE}"
```

Comment:
So, I think these patches allow us to do two things...
- Install kata from sources, without doing any of the other package installs etc. (allows us to check a system has all the dependencies it needs?)
- Allows us to skip installing kata from source (this patch), which then allows us to run all the tests on an already installed system (such as if we are testing packages coming out of OBS?).

If I'm right - maybe it is also time we have a small overview document of what scripts are useful for what situation, and how to configure/invoke them?
The .ci directory is getting quite busy and complex - probably time we started trying to document it a bit.